Peer review edits from Alex (#94)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_deploying-smart-gateways.adoc

done
----

> . Create a new Smart Gateway manifest configuration file from the `/tmp/cloud1-smartgateways.yaml` file. Modify the `metadata.name` and `spec.amqpUrl` fields to include the cloud identifier from your schema. For more information, see <<planning-amqp-address-prefixes_advanced-features>>. To view example Smart Gateway manifests, see <<example-manifests_advanced-features>>.

Author: @leifmadsen This is the main point for you to review please Leif.

Author: Ah ok, thanks. There was a bit of confusion and we were trying to be more explicit in the step. Your suggestion below looks perfect.

> . Create a new Smart Gateway manifest configuration file from the `/tmp/cloud1-smartgateways.yaml` file. Modify the `metadata.name` and `spec.amqpUrl` fields to include the cloud identifier from your schema. For more information, see <<planning-amqp-address-prefixes_advanced-features>>. To view example Smart Gateway manifests, see <<example-manifests_advanced-features>>.

Reviewer: ```suggestion
. Modify the Smart Gateway manifest in the `/tmp/cloud1-smartgateways.yaml` file. Adjust the `metadata.name` and `spec.amqpUrl` fields to include the cloud identifier from your schema. For more information, see <<planning-amqp-address-prefixes_advanced-features>>. To view example Smart Gateway manifests, see <<example-manifests_advanced-features>>.
```
